pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -261,6 +262,13 @@ func (m *manager) Allocate(p *v1.Pod, c *v1.Container) error {
 
 	// Call down into the policy to assign this container CPUs if required.
 	err := m.policy.Allocate(m.state, p, c)
+
+	// If it gets this error it means that the pod requires pod level resources but this is not aligned.
+	// We do not want the pod to fail to schedule on this error so we Admit it, this is just a warning
+	if errors.As(err, &CPUManagerPodLevelResourcesError{}) {
```

Comment:
Rather than handling this case separately, I'd just fold it in to the `if err != nil` case below.

It's best practice to not log when returning an error, so I would change the case below to:

```go
	if err != nil {
		return fmt.Errorf("Allocate error: %w", err)
	}
```